### ISSUE TITLE
Crash when trying to attach

### ISSUE THREAD

**Issue by anaisbetts** [enlight/electron-inspector#3]
```
TypeError: Cannot read property 'ref' of undefined
    at InjectorClient.<anonymous> (C:\Users\paulb\code\tinyspeck\slack-winssb\node_modules\electron-inspector\node_modules\node-inspector\lib\InjectorClient.js:111:22)
    at C:\Users\paulb\code\tinyspeck\slack-winssb\node_modules\electron-inspector\node_modules\node-inspector\lib\DebuggerClient.js:121:7
    at Object.value (C:\Users\paulb\code\tinyspeck\slack-winssb\node_modules\electron-inspector\node_modules\node-inspector\lib\callback.js:23:20)
    at Debugger._processResponse (C:\Users\paulb\code\tinyspeck\slack-winssb\node_modules\electron-inspector\node_modules\node-inspector\lib\debugger.js:95:21)
    at Protocol.execute (_debugger.js:121:14)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:543:20)
```

Trying to attach to Electron 1.4.3, on Windows 64-bit

**Comment by enlight** [comment]
Can also reproduce with Electron 1.4.4 on Win10 x64.

**Comment by Alex-Space** [comment]
Have the same with electron 1.3.1, 1.4.5 (Linux Mint)
![selection_004](https://cloud.githubusercontent.com/assets/7780469/20033103/75dfa0e2-a3c3-11e6-9ff8-640484075960.png)

**Comment by enlight** [comment]
Looks like https://github.com/node-inspector/node-inspector/pull/906 could fix this issue, I'm going to try it out tomorrow. Unfortunately `node-inspector` has been left to rot so this is just one of the many issues that are going to pop up as Chrome keeps updating the DevTools front-end. Once https://github.com/electron/electron/issues/6634 is resolved we won't have to deal with `node-inspector` at all, and `electron-inspector` can be deprecated.

**Comment by dustinblackman** [comment]
@enlight Any news on this one?

**Comment by enlight** [comment]
This should now be fixed in v0.1.4. Note that you'll still see an error in the console but it should no longer bring down the whole front-end. I've added a section to the README to document some of these obvious bugs. Fingers crossed this keeps `node-inspector` limping along for a wee while longer.
